src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -86,6 +86,13 @@ message StreamingMessage {
 
     // Host gets the list of function load responses
     FunctionLoadResponseCollection function_load_response_collection = 32;
+
+    // Host sends required metadata to worker to warmup the worker
```

Comment:
Don't see this change in the [protobuf repo](https://github.com/Azure/azure-functions-language-worker-protobuf).  We should bring the proto changes first to that repo and then consume in other places (host or language workers)

Reply:
Yeah please make sure to follow the protobuf [release process](https://dev.azure.com/msazure/One/_git/AAPT-Antares-Docs?path=/TeamDocs/FunctionTeamDocs/EngineeringProcesses/ReleaseProcesses/ProtobufReleaseProcess.md&version=GBmaster&line=1&_a=preview)

Reply:
Yes, will update to pull changes from proto repo.
PR for protobuf changes: https://github.com/Azure/azure-functions-language-worker-protobuf/pull/84

Reply:
@liliankasem suggested in another PR that we should bring the proto changes to host (after you create a release with the latest proto changes) in a separate PR to be cleaner.

Reply:
Yes, apologies for the delay. I plan on testing the changes on private stamp before releasing the proto changes. 
In process of getting a private stamp as dev deploy is blocked due to some issues